javascript/ql/src/Security/CWE-094/UnsafeDynamicMethodAccess.qhelp

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Invoking a user-controlled method on certain objects can lead to invocation of unsafe functions,
```

Comment:
Invoking/invocation - would it be possible for you to use a synonym for one of the instances to avoid repetition in the same sentence?